setup.cfg

```diff
@@ -1,24 +1,58 @@
 [metadata]
-package_name = ci_watson
+name = ci_watson
 description = CI helper for STScI Jenkins
 author = STScI
 author_email = help@stsci.edu
 license = BSD
+license_file = LICENSE.md
 url = https://github.com/spacetelescope/ci_watson
+long_description = file: README.md
+long_description_content_type = text/markdown
 classifier =
     Framework :: Pytest
     Intended Audience :: Science/Research
     License :: OSI Approved :: BSD License
     Operating System :: OS Independent
     Programming Language :: Python :: 3
+    Programming Language :: Python :: Implementation :: CPython
     Topic :: Software Development :: Testing
     Topic :: Software Development :: Libraries :: Python Modules
 
+[options]
+# We set packages to find: to automatically find all sub-packages
+packages = find:
+include_package_data = True
+zip_safe = False
+setup_requires = setuptools_scm
+install_requires =
+    pytest>=3
+    requests
+python_requires = >=3.5
+
+[options.packages.find]
+exclude =
+    tests
+
+[options.extras_require]
+all =
+    astropy
+test =
+    pytest-astropy-header
+docs =
+    pytest
```

Comment:
`pytest` is already a package dependency, so doesn't need to be a `[docs]` dependency.

Reply:
Okay. I just copy-pasted from the old requirements file. Will remove it.